### PR TITLE
Generate Sphinx docs for publishing to RTD.

### PR DESCRIPTION
**Pull request by changhiskhan**
Main changes:
1. Rename eto.internal to eto._internal so sphinx will skip autodoc
2. Use Autodoc with Napolean to pickup docstrings
3. Use myst-parser to include the README.md contents

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = "Eto Python SDK"
+copyright = "2021, Eto Labs"
```
Comment (eddyxu) [inline docs/source/conf.py:21]: 2022?

File: docs/source/conf.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+author = "Eto Labs"
+
+# The full version, including alpha/beta/rc tags
+release = "0.3.0"
```
Comment (eddyxu) [inline docs/source/conf.py:25]: Do we need to sync this version with the actual codebase?

**Review by eddyxu — APPROVED** [review]
LGTM overall

File: docs/source/conf.py (hunk 2 above)
Comment (changhiskhan) [reply docs/source/conf.py:25]: next version is v0.3 so I was just saving a commit

File: docs/source/conf.py (hunk 1 above)
Comment (changhiskhan) [reply docs/source/conf.py:21]: will fix

**Pull request merged**